.github/workflows/pypi-build-and-deploy.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       # Publish on PyPi
       - name: Publish distribution 📦 to PyPI
         if: startsWith(github.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@release/v1.5.0
+        uses: pypa/gh-action-pypi-publish@v1.5.0
```

Comment:
yeah sorry, I should have done a code suggestion with it - it was confusing since release/v1 and then just v1.5.0